lib/active_merchant/billing/gateways/stripe_payment_intents.rb

```diff
@@ -231,6 +227,23 @@ def store(payment_method, options = {})
         end
       end
 
+      def customer(post, payment, options)
```

Comment:
~~Maybe call this method `add_customer` to match the style of the rest of the class~~
Nevermind, I see that `customer` is actually an endpoint method